src/django_components/apps.py

```diff
@@ -25,7 +26,7 @@ def ready(self) -> None:
         # See https://github.com/EmilStenstrom/django-components/discussions/567#discussioncomment-10273632
         # And https://stackoverflow.com/questions/42907285/66673186#66673186
         if app_settings.RELOAD_ON_TEMPLATE_CHANGE:
-            dirs = get_dirs(include_apps=False)
+            dirs = get_component_dirs(include_apps=False)
```

Comment:
I also renamed `get_dirs` to `get_component_dirs`, so it's purpose is clearer.

Initially I wanted to keep the renaming as a separate MR, but it was already too complicated to untangle the two.